opendihu_examples/isometric_contraction/biceps_muscle/README.md

```diff
@@ -15,6 +15,7 @@ This computes everything in serial and therefore will take a long time. If you c
 ```
 mpirun -n 16 ./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py ramp.py --prestretch_force 10.0
 ```
+Instead of 16 rangs you can also choose any other number.
```

Comment:
Good to mention that 16 is not the only option, but not every number is possible!
```suggestion
Instead of 16 ranks you can also choose any other number.
```